tests/unit/helpers/questionnaires.test.js

```diff
@@ -190,7 +190,7 @@ describe('getUserQuestionnaires', () => {
     );
   });
 
-  it('should return an empty array if no expectations questionnaire', async () => {
+  it('should return an empty array if is started and no expectations questionnaire', async () => {
```

Comment:
not started

Reply:
si elle est commencé la question ne se pose pas pour le questionnaire des attentes

Reply:
done